source/support.tex

```diff
@@ -3974,10 +3982,8 @@
     static const weak_equality nonequivalent;
 
     // comparisons
-    friend constexpr bool operator==(weak_equality v, @\unspec@) noexcept;
-    friend constexpr bool operator!=(weak_equality v, @\unspec@) noexcept;
-    friend constexpr bool operator==(@\unspec@, weak_equality v) noexcept;
-    friend constexpr bool operator!=(@\unspec@, weak_equality v) noexcept;
+    friend constexpr bool operator==(weak_equality v, unspecified) noexcept;
```

Comment:
`unspecified` should have remained `@\unspec@`.

Reply:
Fixed.